db/migrate/20150312234511_add_personal_atts_to_user.rb

```diff
@@ -0,0 +1,18 @@
+class AddPersonalAttsToUser < ActiveRecord::Migration
+  def up
+    add_column :users, :name, :string
+    add_column :users, :profile_pic_url, :string
+    add_column :users, :language, :string
```

Comment:
A lot of these are cute but purely cosmetic. I'm not opposed to removing all of them, but language just doesn't have any point.